cmd/cmd.go

```diff
@@ -117,41 +118,42 @@ var (
 				case config.Mysql:
 					listener, err := listener.NewMysqlListener(listenerConf)
 					if err != nil {
-						panic(err)
+						log.Fatalf("create mysql listener failed %v", err)
 					}
 					dbListener := listener.(proto.DBListener)
 					executor := executors[listenerConf.Executor]
 					if executor == nil {
-						panic(errors.Errorf("executor: %s is not exists for mysql listener", listenerConf.Executor))
+						log.Fatalf("executor: %s is not exists for mysql listener", listenerConf.Executor)
```

Comment:
If a fatal error accounted, maybe we should stop progress going on.

Reply:
this func, after print log, will call os.Exit

Reply:
Got it.